Unity/2019.1/Events with Caliper/README.md

```diff
@@ -0,0 +1,27 @@
+﻿I am following the instructions from the [caliper-net example](https://github.com/IMSGlobal/caliper-net#getting-started).
```

Comment:
I see a lot of file changes here which some are made by unity as part of creating components in Unity. I think listing what files type are default created by Unity, what type is generated by when creating object/component for test cases in unity and files created for coding (like `.cs') files will be useful.

Reply:
I added a list of files i created to the readme